Index: dwm/dwm/config.def.h
===================================================================
--- dwm.orig/dwm/config.def.h	2013-10-21 08:51:46.690960758 +0200
+++ dwm/dwm/config.def.h	2013-10-21 08:52:25.026961814 +0200
@@ -71,6 +71,7 @@
 #include "push.c"
 #include "focusmaster.c"
 #include "shifttags.c"
+#include "keepfloatingposition.c"
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
