--- dwm.orig/config.def.h	Sat Feb 11 20:58:23 2012 +0100
+++ dwm/config.def.h	Sat Feb 11 20:59:19 2012 +0100
@@ -8,8 +8,8 @@
 static const char selbordercolor[]  = "#ff0000";
 static const char selbgcolor[]      = "#005577";
 static const char selfgcolor[]      = "#eeeeee";
-static const char floatnormbordercolor[] = "#005577";
-static const char floatselbordercolor[]  = "#005577";
+static const char floatnormbordercolor[] = "#885599";
+static const char floatselbordercolor[]  = "#ff0000";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const Bool showbar           = True;     /* False means no bar */
