# HG changeset patch
# Parent ca845c5c0a7bff71030b46813f83fe10e96533c0

Index: dwm/config.def.h
===================================================================
--- dwm/config.def.h.orig
+++ dwm/config.def.h
@@ -87,6 +87,7 @@ static const char *termcmd[]  = { "x-ter
 #include "zoomswap.c"
 #include "centredmaster.c"
 #include "push.c"
+#include "focusmaster.c"
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
@@ -156,6 +157,7 @@ static Key keys[] = {
 	{ MODKEY|ShiftMask,             XK_d,      moveresize,     {.v = "0x 0y 10w 0h" } },
 	{ MODKEY|ShiftMask,             XK_s,      moveresize,     {.v = "0x 0y 0w 10h" } },
 	{ MODKEY|ShiftMask,             XK_w,      moveresize,     {.v = "0x 0y 0w -10h" } },
+	{ MODKEY|ControlMask,           XK_Return, focusmaster,    {0} },
 };
 
 /* button definitions */
