# HG changeset patch
# Parent 6d2da43e7b8a81af13086a26b63b9b008288d5ef

Index: dwm/config.def.h
===================================================================
--- dwm.orig/config.def.h
+++ dwm/config.def.h
@@ -65,6 +65,7 @@ static const char *termcmd[]  = { "x-ter
 #include "moveresize.c"
 #include "tagall.c"
 #include "maximize.c"
+#include "zoomswap.c"
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
